### ISSUE TITLE
JS: "Uncaught TypeError: Cannot read property 'value' of null" with destructuring assignment and trailing comma

### ISSUE THREAD

**Issue by sergionsz** [nikhilkalige/docblockr#219]
With code like:

```
const ArrowFunc = ({
  param1,
  param2,
}) => do_something_with(param1, param2);
```

dockblockr throws the error:

```
Uncaught TypeError: Cannot read property 'value' of null
```

with the stack trace:

```
TypeError: Cannot read property 'value' of null
    at DocsParser.JsParser.get_arg_name (/Users/sernavar/.atom/packages/docblockr/lib/languages/javascript.js:128:16)
    at DocsParser.parse_args (/Users/sernavar/.atom/packages/docblockr/lib/docsparser.js:265:48)
    at DocsParser.format_function (/Users/sernavar/.atom/packages/docblockr/lib/docsparser.js:130:32)
    at DocsParser.parse (/Users/sernavar/.atom/packages/docblockr/lib/docsparser.js:44:37)
    at DocBlockrAtom.module.exports.DocBlockrAtom.parse_command (/Users/sernavar/.atom/packages/docblockr/lib/docblockr-worker.js:287:29)
    at atom-workspace.<anonymous> (/Users/sernavar/.atom/packages/docblockr/lib/docblockr-worker.js:77:16)
    at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:260:29)
    at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
    at KeymapManager.module.exports.KeymapManager.dispatchCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:580:16)
    at KeymapManager.module.exports.KeymapManager.handleKeyboardEvent (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-keymap/lib/keymap-manager.js:388:22)
    at WindowEventHandler.module.exports.WindowEventHandler.handleDocumentKeyEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:98:36)
    at HTMLDocument.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/src/window-event-handler.js:3:61)
```

A message is shown referencing issue #197. However, the issue describes a very different scenario that also throws the same error message.

dockblockr version: 0.8.5

Please let me know if you need more information.

**Comment by MoritzKn** [comment]
Yes, this error occurs every time there is something wrong with parsing function arguments.  
The root of your problem is that destructuring assignment is not yet supported. See: #180.
In any way this shouldn't result in an uncaught error.

**Comment by MoritzKn** [comment]
I will close this issue, because the problem will be addressed in #180
